### PR TITLE
Remove `kuberun` command

### PR DESCRIPTION
**Pull request by bentsherman**

### CODE REVIEW

File: docs/cli.rst [hunk 1]
```diff
@@ -728,6 +728,9 @@ Display information about a specific project::
 kuberun
 --------------------
 
+.. danger::
+  The ``kuberun`` command was removed in version 22.04. Consider the use of Fusion or Nextflow Tower as a better alternative.
```
Comment (bentsherman) [inline docs/cli.rst:732]:
```suggestion
  The ``kuberun`` command was removed in version 23.04. Consider the use of Fusion or Nextflow Tower as a better alternative.
```

Comment (bentsherman) [inline docs/cli.rst:732]:
```suggestion
  The ``kuberun`` command was removed in version 23.04. Consider the use of Fusion or Nextflow Tower as a better alternative.
```

**Comment by pditommaso** [comment]
Maybe there has been a misunderstanding, my suggestion was to not implement it in CLI v2 but keep in the current CLI (!). does it makes sense?

**Comment by bentsherman** [comment]
I thought you said you wanted to remove it outright

**Comment by pditommaso** [comment]
bit too harsh!

**Comment by bentsherman** [comment]
Lol. Another day then

**Pull request closed without merging**